plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/util/TestCassandraClusteringPredicatesExtractor.java

```diff
@@ -61,7 +62,7 @@ public void testBuildClusteringPredicate()
                         col1, Domain.singleValue(BIGINT, 23L),
                         col2, Domain.singleValue(BIGINT, 34L),
                         col4, Domain.singleValue(BIGINT, 26L)));
-        CassandraClusteringPredicatesExtractor predicatesExtractor = new CassandraClusteringPredicatesExtractor(cassandraTable.getClusteringKeyColumns(), tupleDomain, cassandraVersion);
+        CassandraClusteringPredicatesExtractor predicatesExtractor = new CassandraClusteringPredicatesExtractor(new CassandraTypeManager(), cassandraTable.getClusteringKeyColumns(), tupleDomain, cassandraVersion);
```

Comment:
nit: Replace `new CassandraTypeManager()` with `CASSANDRA_TYPE_MANAGER`.

Reply:
I made the mistake of replacing it in the 3rd commit while I should have replaced it in the 1st commit.. Just fixed :)